src/index.js

```diff
@@ -29,6 +31,40 @@ export default function supernova(env) {
       const element = useElement();
       const rect = useRect();
       const Theme = useTheme();
+      const [linked, setLinked] = useState(false);
+      const selectionsAPI = useSelections();
+      useAction(
+        () => ({
+          action() {
+            setLinked(!linked);
+          },
+          icon: {
+            d: 'svg path',
+          },
+        }),
+        [linked]
+      );
```

Comment:
This should create a button on the selection toolbar. However not supported by Nebula yet. The svg path needs to be changed of course to the icon that we are going to use